coding-for-reusability/Exercise3/solution/Solution.ts

```diff
@@ -0,0 +1,19 @@
+type IdentityType = string | number
+
+interface FormattedIdentity<ID> {
+    name: string, 
+    identifier: ID
+}
+
+function formatIdentity<ID extends IdentityType>(value: string, id: ID): FormattedIdentity<ID> {
+    return {
+        name: value,
+        identifier: id
+    }
+}
+
+formatIdentity('Christina', 12345)
+formatIdentity<string>('Christina', "12345")
+formatIdentity('Christina', { id: 12345})
```

Comment:
I know we probably can’t use a linter on these files because most of them intentionally have errors, but I wonder how we could just make our formatting consistent.